lib/generator.dart

```diff
@@ -35,7 +35,9 @@ main(List<String> args) {
   var entryPoint = args[1];
   var classAnnotations = args[2].split(',')..addAll(_DEFAULT_INJECTABLE_ANNOTATIONS);
   var output = args[3];
-  var packageRoots = (args.length < 5) ? [Platform.packageRoot] : args.sublist(4);
+  var packageRoots = (args.length < 5)
+      ? [path.fromUri(Platform.packageRoot ?? '')]
```

Comment:
Yep we'll need to bump the SDK constraint to `>=1.13.0 <2.0.0`. Should be close to a no-op since we're already picking up that same constraint via analyzer 0.27.